docs/plugin/plugin.md

```diff
@@ -27,43 +27,260 @@ Options:
+
+- `--namespace=risingwave` Namespace of risingwave instance. If not set, select the default namespace.
+
+### RisingWave Instance List
+
+Command: `kubectl rw list INSTANCE_NAME [options]`
+
+Examples:
+
+- List all clusters and sync to local config.
+
+`kubectl rw list`
+
+- Filter by namespace.
+
+`kubectl rw list --namespace=foo`
+
+- Get risingwave instances by selector.
+
+`kubectl rw list -l foo=bar`
+
+Options:
+
+- `-A, --all-namespaces=false` Whether list instances in all namespaces.
+
+- `-l, --selector=''` Selector (label query) to filter on, not including uninitialized ones, supports '=', '==', and '!='. (e.g. -l key1=value1,key2=value2)
+
+- `-n, --namespace='default'` Namespace of risingwave instance. If not set, select the default namespace.
+
+### RisingWave Instance Describe
+
+Command: `kubectl rw describe INSTANCE_NAME [options]`
+
+Describe a risingwave instance.
+
+Examples:
+
+- Describe risingwave named example-rw.
+
+`kubectl rw describe example-rw`
+
+- Describe risingwave instance named example-rw in namespace foo.
+   
+`kubectl rw describe example-rw -n foo`
+
+Options:
+
+- `-n, --namespace='default'` Namespace of risingwave instance. If not set, select the default namespace.
+
+- `-c, --choice='spec'` The section of the risingwave instance you would like to describe. Spec, status and all are the only valid values.
+
+## Deployment commands
+
+### Scale
+
+Command: `kubectl rw scale INSTANCE_NAME [options]`
+
+Scale a risingwave Instance.
+
+Examples:
+
+- Scale compute-node of the risingwave named example-rw to 2.
+  
+`kubectl rw scale example-rw -t 2`
+
+- Scale frontend of the risingwave named example-rw to 2 in the foo namespace.
+
+`kubectl rw scale example-rw -n foo -c frontend -t 2`
+
+- Scale frontend of the risingwave which named example-rw to 2 and in the foo namespace and in the test group.
+
+`kubectl rw scale example-rw -n foo -c frontend -t 2 -g test`
+
+Options:
+
+- `-c, --component='compute-node'` The component which you want to scale.
+
+- `-g, --group='default'` The group name of the component. If not set, scale the default group
+
+- `-t, --target=-1` The target number.Describe risingwave named example-rw.
 
-example: `kubectl rw delete example -n risingwave`
+- `-n, --namespace='default'` Namespace of risingwave instance. If not set, select the default namespace.
+
+### Stop
+
+Command: `kubectl rw stop INSTANCE_NAME [options]`
+
+Stop a risingwave instance.
+
+Examples:
+
+- Stop risingwave named example-rw in default namespace.
+  
+`kubectl rw stop example-rw`
+
+- Stop risingwave named example-rw in foo namespace.
+
+`kubectl rw stop example-rw -n foo`
 
 Options:
 
-- `--namespace=risingwave`
+- `-n, --namespace='default'` Namespace of risingwave instance. If not set, select the default namespace.
 
-#### RisingWave Instance List
+### Restart
 
-Command: `kubectl minio tenant info TENANT_NAME [options]`
+Command: `kubectl rw restart INSTANCE_NAME [options]`
```

Comment:
Do we support restarting the Pods of the specified components? E.g., I just want to restart the frontend and the compute nodes.

Reply:
Not at the moment, but I will create an issue to add that feature!